awx/api/views/__init__.py

```diff
@@ -578,8 +578,8 @@ class ScheduleZoneInfo(APIView):
     swagger_topic = 'System Configuration'
 
     def get(self, request):
-        zones = [{'name': zone} for zone in models.Schedule.get_zoneinfo()]
-        return Response(zones)
+        zone_info = models.Schedule.get_zoneinfo_with_links()
+        return Response(zone_info)
```

Comment:
Does this change the returned JSON schema of the view?

Reply:
Interestingly, I don't _think_ this view is captured by the api-schema check. We could add it to the validated schema by writing a unit test that made a GET to the view. This is an unusual case where that may actually be helpful (unless the time zone library changes frequently).